### DISCUSSION TITLE
Missing frames? / buffering?

### DISCUSSION THREAD

**Discussion by JA16122000** [mrlt8/docker-wyze-bridge#623] [Q&A]
very often, both streams are "hanging" and they only go further, when i refresh my page.
The left one, stops and keep hanging in the next screenshot:

https://imgur.com/a/iNFpRve

Here some other clips:

https://imgur.com/a/TkJoXAq

And 

https://imgur.com/a/eFZhjXd

**Comment by mrlt8** [comment]
This seems like something we might be able to tweak in the videojs config.

**Comment by JA16122000** [comment]
Maybe this is also the reason, why my 15 minutes files, actually 12 or 13 minutes are.

**Reply by JA16122000** [reply]
I noticed, when i put this in the YML file, by env.
- H264_ENC=h264_v4l2m2m

Then the recordings, ar almost nice 15 minutes.
So maybe, somehow this is what i've been missing?

**Reply by mrlt8** [reply]
hmm, not sure about that. h264_v4l2m2m will actually re-encode the video, so there's a possibility that the frame rate is off?

**Comment by JA16122000** [comment]
It looks like, my recordings are only 7 minutes :O

**Reply by mrlt8** [reply]
are you missing some minutes or is the playback speed wrong?

**Reply by JA16122000** [reply]
Every new "15 minutes" recording, keeps getting worse.
after every 2 seconds, it missing 1 second.
00:01 00:02 -jumps to- 00:04 00:05 -jumps to- 00:07 00:08 -jumps to- 00:10 etc etc etc

https://imgur.com/GjsHCsv

**Comment by JA16122000** [comment]
It is only on record files.
They are all shorther then 15 minutes.
Some ar 7 minutes, some 8, 9, 11, 12, 13.
They are all missing frames in every minute.
In every minute, the time is missing some frames / seconds.

While watching the live stream, there seems no big problems.

Also on nightview (live) time is being much faster

**Reply by mrlt8** [reply]
hmm, we're using the [tee muxer ](https://ffmpeg.org/ffmpeg-formats.html#tee) in ffmpeg so the recordings should be identical to what you see in the other streams. 
I'll have to double check some of the ffmpeg commands we're using as some of them may be dropping frames to keep up to realtime and/or frames received out of order.

**Reply by JA16122000** [reply]
Double check would be nice.
It is keep getting worse.
Am i the only one who is missing frames?
Output movies are far from 15 minutes

**Reply by mrlt8** [reply]
Does `KEEP_BADFRAMES=true` make any difference?

**Reply by JA16122000** [reply]
Nope, that doesn't make any difference at all

**Comment by JA16122000** [comment]
@mrlt8 This is getting worse.
Some files are 7 minutes :(

**Comment by JA16122000** [comment]
![ok](https://user-images.githubusercontent.com/112784196/218135670-84fb482c-9008-49c3-91de-9a0c9eb40aa3.png)

**Comment by JA16122000** [comment]
@mrlt8 
Can i test a "arm" build? [This](https://github.com/mrlt8/docker-wyze-bridge/issues/294#issuecomment-1043740792)
`        build:`
`            context: ./app`
`            dockerfile: Dockerfile.armv6`

context: ./app wont work

**Comment by yeeahnick** [comment]
Old post but did you ever figure this out? Mine started doing the same out of the blue. It was perfect and now it is not.

**Reply by JA16122000** [reply]
Nope.

After a few updates @mrlt8 did, this problem was gone
